### PR TITLE
fix(Studies): qidoSupportsIncludeField should be true by default

### LINKED ISSUE #772: Catch repo up to latest ohif-core source
This issue got caught up in our migration and needs ported: https://github.com/OHIF/ohif-core/issues/63#event-2529931140

### PR DESCRIPTION
**Pull request by jfmedeiros1820**
fix #772

### CODE REVIEW

File: platform/core/src/studies/services/qido/studies.js [hunk 1]
```diff
@@ -118,9 +118,13 @@ export default function Studies(server, filter) {
   };
 
   const dicomWeb = new api.DICOMwebClient(config);
+  let serverSupportsQIDOIncludeField = server.qidoSupportsIncludeField;
```
Comment (dannyrb) [inline platform/core/src/studies/services/qido/studies.js]:
```suggestion
 server.qidoSupportsIncludeField = server.qidoSupportsIncludeField === undefined ? true : server.qidoSupportsIncludeField;
```

If possible, I think we can avoid creating a new var, and just update value if it's undefined.

Comment (dannyrb) [reply platform/core/src/studies/services/qido/studies.js]: Looks like GitHub doesn't support multi-line suggestions. Replacing all changes with something like:

```js
server.qidoSupportsIncludeField = server.qidoSupportsIncludeField === undefined 
    ? true
    : server.qidoSupportsIncludeField;
```

Comment (omartoutounji) [reply platform/core/src/studies/services/qido/studies.js]: Added your suggestions to https://github.com/OHIF/Viewers/pull/801

**Comment by dannyrb** [comment]
Fixed here: https://github.com/OHIF/Viewers/pull/801

**Pull request closed without merging**